front-end-master/about_us/about.html

```diff
@@ -22,9 +22,49 @@
 
 
 <body>
+	<div class="pop-up-container pop-up-hidden">
+		<div id="pop-up-window" class="pop-up">
+			<img src="icons/cross.svg" class="cross">
+			<div class="profile-data">
+				<img src="members/intekhab_hossain.jpg">
+				<div class="info">
+					<span class="name">Intekhab Hossain</span>
+					<span class="designation">Founder and Statistician</span>
+					<span class="project-name">The R-Squared Project</span>
+				</div>
+				<div class="social-links">
+					<a href="#"><img src="icons/email.svg"></a>
```

Comment:
We should remember to fill these out before launching

Reply:
yes I'll make a function to get the values from the person card and fill out these automatically.

Reply:
btw, have any idea on what to do after the user clicks on the 'mail' icon? is there any way to take the user to their mail client, open a compose mail window and automatically write the person's email id to the compose mail window 'To' section. I've seen mentions of the mailto thing but not really sure how to use it and some says it's not reliable and can be blocked by adblockers and related extensions. can you help me with this?

Reply:
Yes. Its a bit tricky and would require a bit of brainstorming. Don't want to rush this. For now, I think we should just have the email as plan text. Any other enhancement can be looked into and done later.